src/main/java/io/cdap/plugin/http/source/common/HttpErrorHandlingStrategy.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.http.source.common;
+
+/**
+ * Indicates error handling strategy which will be used to handle unexpected http status codes.
+ */
+public enum HttpErrorHandlingStrategy implements EnumWithValue {
+  SUCCESS("Success"),
```

Comment:
what does success mean?

Reply:
Means that a certain http code. Will not be treated as an error. But as fully valid page. Example:
`2..:Success, .*:Fail`